packages/cozy-notifications/src/style.css

```diff
@@ -0,0 +1,102 @@
+h1, h2, h3, h4, h5, h6 {
+  display:block;
+  margin:0;
+  padding:0;
+}
+
+.tableRow {
+  border-bottom: 1px #d6d8da solid;
```

Comment:
there is no way to at least use the UI palette?

Reply:
Can't we add new component to our cozy-mjml lib?

Reply:
cozy-mjml is not structured as a lib for now, its entrypoint is a binary. I agree that we should probably merge the two libs, I am not sure if it is the right time though.

I am going to try to see if it is easy to add a new component but since the original goal of this PR was to extract the code from banks to be able to use it in konnectors, I think that if I do not succeed quickly, I will postpone this task. Would you be ok with that ?

Reply:
I didn't know that this lib was taken from banks ! 

So let's not use cozy-mjml for now, but is it possible to harmonize this lib? For instance it seems that in our exemple here our cozy-logo is 100px width, and there https://github.com/cozy/cozy-libs/blob/master/packages/cozy-mjml/src/components/MJHeader.js#L19 it seems to be 129px ?

Reply:
I fixed the example :)

Reply:
> there is no way to at least use the UI palette?

Done in https://github.com/cozy/cozy-libs/pull/807/commits/ba7e1dc7e614ad2876d0c8bf00bc9a726b4f623f